src/main/java/cz/tstrecha/timetracker/service/impl/TaskServiceImpl.java

```diff
@@ -37,5 +41,13 @@ public TaskDTO createEmptyTask(IdentifierType identifierType, String identifierV
         return taskMapper.toDTO(taskEntity);
     }
 
+    public List<TaskDTO> searchInTasks(Long limit, String query, LoggedUser loggedUser) {
+        if (query.length() < Constants.MIN_SEARCH_LENGTH){
+            return List.of();
+        }
+        query = StringUtils.strip(query);
```

Comment:
Updating param is not wrong, but also not best practice -> its sort of unintuitive and can lead to some really bad reference errors something like:

`var databaseSearchQuery = "%" + StringUtils.strip(query) + "%";`